packages/core/src/routes/admin-user.ts

```diff
@@ -43,9 +47,12 @@ export default function adminUserRoutes<T extends AuthedRouter>(router: T) {
       async () => {
         const search = parseSearchParamsForSearch(searchParams);
         const hideAdminUser = isTrue(searchParams.get('hideAdminUser'));
-
+        const excludeRoleId = searchParams.get('excludeRoleId');
+        const excludeUsersRoles = excludeRoleId ? await findUsersRolesByRoleId(excludeRoleId) : [];
+        const excludeUserIdsByRole = excludeUsersRoles.map(({ userId }) => userId);
```

Comment:
`hideAdminUser` can be combined into the new logic?

Reply:
Yes, after we remove `roleNames`, created LOG-5137 to track